puppet/modules/logfile/manifests/init.pp

```diff
@@ -0,0 +1,26 @@
+#
+# make syslog log to a particular file for a particular process.
+#
+
+define logfile($process=$title) {
+  $logfile = "/var/log/leap/${title}.log"
+
+  rsyslog::snippet { "50-${name}":
+    content => 'if $programname startswith "${process}" then "${logfile}"
+&~'
```

Comment:
also, there is still 99-tapicero.conf and possible other old rsyslog configs in /etc/rsyslog.d, they need to get purged by puppet.